src/types.jl

```diff
@@ -2846,6 +2846,15 @@ for R in (
     @eval $R(sizehint::Integer=0) = $R(map(T -> sizehint!(T(), sizehint), fieldtypes($R))...)::$R
 end
 
+function Base.isempty(net::T) where {T <: Network}
+    for i in 1:fieldcount(T)
+        if typeof(getfield(net, i)) <: Records && !isempty(getfield(net, i))
```

Comment:
what do you think of either
```suggestion
        x = getfield(net, i)
        if x isa Records && !isempty(x)
```
or
```suggestion
        if fieldtype(net, i) <: Records && !isempty(getfield(net, i))
```
?

Reply:
Using `fieldtype` sounds good, thanks!